staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/azure_test.go

```diff
@@ -125,8 +126,8 @@ func newFackeAzureToken(accessToken string, expiresOn string) adal.Token {
 		AccessToken:  accessToken,
 		RefreshToken: "fake",
 		ExpiresIn:    "3600",
-		ExpiresOn:    expiresOn,
-		NotBefore:    expiresOn,
+		ExpiresOn:    json.Number(expiresOn),
+		NotBefore:    json.Number(expiresOn),
```

Comment:
Will old clients work with new servers and vice versa?

Reply:
yes, `json.Number` is actually string:
```
// A Number represents a JSON number literal.
type Number string
```
https://github.com/golang/go/blob/914a25c0eb8aacdbb5cddd3e7d5e6ee7b019f88f/src/encoding/json/decode.go#L187